### DISCUSSION TITLE
Multiple Selections in Editor

### DISCUSSION THREAD

**Discussion by superloud-io** [dbeaver/dbeaver#18114] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Making the same edit many lines of SQL is difficult. For example, if you want to add a space after a comma for 100 lines, you have to and space 100 times. Having a multi-select option (multiple cursors), you'd only need to click 100 times and press space once, saving a lot of time.

**Describe the solution you'd like**
It would be great to have a multi-select option like other editors so that one can edit many lines simultaneously. For example, in Sublime, if you hold Command while selecting lines, you can make multiple selections.

**Describe alternatives you've considered**
Currently, I'll copy and paste my code into another editor, make my changes, then paste it back into DBeaver. I have looked at other free editors, but they do not have this functionality.

There was an attempt at one point to provide this feature (https://github.com/dbeaver/dbeaver/issues/6064) but it missed the mark and does do anything like what was asked for in the original post.

**Additional context**
Here is an example of the functionality in Sublime:
![2022-10-24 18 18 57](https://user-images.githubusercontent.com/68535528/197641062-1f902843-f42a-4d24-b372-e1e2945ce215.gif)

**Comment by flambchap** [comment, marked as answer]
Hi. You can do that in DBeaver already by using `Alt + Click` on the place where you want to see an additional caret: https://www.eclipse.org/eclipse/news/4.22/platform.php#multi-selection.
